craft_store/store_client.py

```diff
@@ -177,6 +180,8 @@ def login(
         # Save the authorization token.
         self._auth.set_credentials(store_authorized_macaroon)
 
+        return store_authorized_macaroon
```

Comment:
You should encode auth stuff before returning it to the wild (and decode it when getting from the envvar)

Reply:
https://github.com/canonical/craft-store/pull/18/commits/b39644184ad528d9aa54b36c25a4f10d01b5b8a5